### DISCUSSION TITLE
quarto file with shiny in a quarto project

### DISCUSSION THREAD

**Discussion by ManuelSpinola** [quarto-dev/quarto-cli#944] [Q&A] [answered]
Does quarto file with shiny work in a quarto project with a yaml?

I tried to run the Old Faithful example but I got the error:

ERROR: Internal Error: getYamlIntelligenceResource called with missing resource schema/project.yml

**Comment by cscheid** [comment, marked as answer]
This was an oversight on our part. We fixed it in 4186c2219 - thanks for reporting!
